youtube_dl/extractor/joj.py

```diff
@@ -104,5 +110,5 @@ def _real_extract(self, url):
             'title': title,
             'thumbnail': thumbnail,
             'duration': duration,
-            'formats': formats,

```

Comment:
The code typically leaves a trailing `'` in lists, dicts, tuples (even when not required) as it's valid syntax and avoids having to change two lines to insert a new line. You could leave this as-is.